<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>欢迎使用中国电信精品渠道终端销售系统</string>
  </property>
  <widget class="QStackedWidget" name="login_stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>481</height>
    </rect>
   </property>
   <widget class="QWidget" name="login_page">
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>手机登录</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>账号登录</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>扫码登录</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="login_style_stackedWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>601</width>
       <height>401</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="account_login_page">
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>91</x>
         <y>50</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>服务器：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>50</y>
         <width>301</width>
         <height>22</height>
        </rect>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>150</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>登录密码：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>150</y>
         <width>301</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>100</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>登录账号：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="account_comboBox">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>100</y>
         <width>301</width>
         <height>22</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="checkCode_lineEdit">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>200</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>67</x>
         <y>200</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>图形验证码：</string>
       </property>
      </widget>
      <widget class="QCommandLinkButton" name="account_login_commandLinkButton">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>290</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>250</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>记住账号</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>250</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>记住密码</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>250</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>显示密码</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>190</y>
         <width>100</width>
         <height>35</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>35</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="phone_login_page">
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>78</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>手机号：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>74</y>
         <width>271</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>96</x>
         <y>125</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>图形验证码：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>125</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>126</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>图形验证码</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>95</x>
         <y>175</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>短信验证码：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>177</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>175</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>获取短信验证码</string>
       </property>
      </widget>
      <widget class="QCommandLinkButton" name="phone_login_commandLinkButton">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>250</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="scan_login_page">
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>300</y>
         <width>211</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>提示文字：当前验证码将在60秒后过期</string>
       </property>
      </widget>
      <widget class="QrcodeGenerate" name="qrcode_widget" native="true">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>250</width>
         <height>250</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>250</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="findPassword_page"/>
   <widget class="QWidget" name="resetPassword_page"/>
   <widget class="QWidget" name="loginSettings_page"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QrcodeGenerate</class>
   <extends>QWidget</extends>
   <header location="global">qrcodegenerate.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>radioButton</tabstop>
  <tabstop>radioButton_2</tabstop>
  <tabstop>radioButton_3</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>account_comboBox</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>checkCode_lineEdit</tabstop>
  <tabstop>account_login_commandLinkButton</tabstop>
  <tabstop>checkBox_3</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>checkBox_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
